README.md

```diff
@@ -517,6 +517,15 @@ not supported to write them to individual databases. The first point written
 defines the database, other values are silently ignored!
 
 
+## Contributing
+
+##### Custom influxdb test connection
+
+```
+export INSTREAM_HOST=localhost
+export INSTREAM_HTTP_PORT=8086
```

Comment:
Looking at the variable names using exports might mess with a generic setup on the machine. But other than a note about predefined testing environment variables in the `### Connections` section (referring to the testing section of the README) there should be nothing to worry about.